src/content/en/2020/markup.md

```diff
@@ -80,8 +80,7 @@ A page's document size refers to the amount of HTML bytes transferred over the n
 * The largest document by far weighs 64.16 _MB_, almost deserving its own analysis and chapter in the Web Almanac.
 
 {# TODO(analysts): Should 25,237 bytes be divided by 1000 or 1024 to convert to KB? 1000 seems to be used here but most chapters use 1024. Are the stats above also off? #}
```

Comment:
@Tiggerito?

Reply:
I agree, 1024 should be used. I'm not confident on the above stats. I'd like to use an example we can prove. I think the problem was the bytesHtmlDoc which I think it more true to what we would call the document got cut off at 16,777,215. 

I used Screaming Frog to crawl all the ones in the list, and double checked the big ones in Chrome. Drum roll...

https://www.linkshops.com/ contains a js bundle that is 28.1MB.

https://www.boonterm.com/web/index1.php has 22.8MB of html (embedded images) and references a 34.7MB mp4 file.

https://www.aci.edu.sg/ has 33.7MB of html and took me over a minute to load. Also embedded images.